README.md

```diff
@@ -16,10 +16,13 @@ FluidSynth works on all major platforms, so pyFluidSynth should also.
 
 FluidSynth 1.0.7 (or later version)
```

Comment:
Is this the version currently in use?

Reply:
Not afaik, but that version# was there before - I just updated the link to fluidsynth's website.